Sources/python/SimEx/Utilities/wpg_to_opmd.py

```diff
@@ -0,0 +1,88 @@
+import sys
+import numpy
+import math
+import h5py
+import os
+
+from SimEx.Utilities import OpenPMDTools as opmd
+
+def _convertToOPMD(h5_in):
+    """ Take native wpg output and rewrite in openpmd conformant way. """
+
+    # Load the generated h5 file.
+    #wpg_outputs = [os.path.join( self.output_path, f) for f in os.listdir( self.output_path ) ]
+    wpg_outputs = [h5_in]
+    wpg_outputs = [f for f in wpg_outputs if f.split('.')[-1] == 'h5']
+    print wpg_outputs
+
+    for i,f in enumerate(wpg_outputs):
+        h5 = h5py.File( f, 'r')
+        opmd_h5 = h5py.File(f+'.opmd', 'w')
+
+        # Get number of time slices in wpg output.
+        data_shape = h5['data/arrEhor'].value.shape
+        number_of_x_meshpoints = data_shape[0]
+        number_of_y_meshpoints = data_shape[1]
+        number_of_time_steps = data_shape[2]
+
+        for it in range(number_of_time_steps):
```

Comment:
offline result: no that would be a wrong assumption. the planes are at a specific point in space and information are given for those over time steps t.